src/HDF5.jl

```diff
@@ -931,7 +931,13 @@ function g_create(f::Function, parent::Union{HDF5File, HDF5Group}, args...)
     end
 end
 
-function d_create(parent::Union{HDF5File, HDF5Group}, path::String, dtype::HDF5Datatype, dspace::HDF5Dataspace, lcpl::HDF5Properties, dcpl::HDF5Properties, dapl::HDF5Properties, dxpl::HDF5Properties)
+# N.B. lcpl does not have a default to distinguish it from the no-keyword method of the
```

Comment:
```suggestion
# NOTE: lcpl does not have a default to distinguish it from the no-keyword method of the following method.
```
Maybe we should also not that this is added only for deprecations purposes and can be removed in the next minor version bump?